src/Silverback.Integration.Kafka/Messaging/KafkaConsumerEndpoint.cs

```diff
@@ -366,6 +364,10 @@ private void Init(string[] topicNames, KafkaClientConfig? clientConfig)
             Names = topicNames;
 
             Configuration = new KafkaConsumerConfig(clientConfig);
+            if (string.IsNullOrWhiteSpace(Configuration.GroupId))
```

Comment:
Ah...here, I see. Then why not do it only here? (Even though I'm not sure that the `Validate` method is the proper place.)

Reply:
No longer needed, see commit: https://github.com/BEagle1984/silverback/pull/172/commits/491926231cbcd0b64afe9452e05c97e0fe55ff05